streams/src/test/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyKeyValueStoreTest.java

```diff
@@ -171,6 +208,10 @@ public void shouldThrowInvalidStoreExceptionDuringRebalance() throws Exception {
         rebalancing().get("anything");
     }
 
+    @Test(expected = InvalidStateStoreException.class)
+    public void shouldThrowInvalidStoreExceptionOnApproximateNumEntriesDuringRebalance() throws Exception {
+        rebalancing().approximateNumEntries();
```

Comment:
Theoretically yes, it can throw, it is a good point. I was a bit confused as other similar tests didn't splitted this into another `try-catch`. So do you think we should also change other tests with `rebalancing()` call?